package.json

```diff
@@ -65,14 +65,20 @@
     "unlink-all": "wsrun unlink",
     "hash": "wsrun --exclude-missing hash",
     "copy-css": "copy-and-watch \"packages/base/dist/css/**/*\" packages/main/dist/resources/css/base/ && copy-and-watch \"packages/theming/dist/css/**/*\" packages/main/dist/resources/css/theming/ && copy-and-watch \"packages/main/dist/css/**/*\" packages/main/dist/resources/css/main/ && copy-and-watch \"packages/base/dist/css/**/*\" packages/fiori/dist/resources/css/base/ && copy-and-watch \"packages/theming/dist/css/**/*\" packages/fiori/dist/resources/css/theming/ && copy-and-watch \"packages/main/dist/css/**/*\" packages/fiori/dist/resources/css/main/",
-    "prerelease": "node ./.github/actions/pre-release.js"
+    "prerelease": "node ./.github/actions/pre-release.js",
+    "prepare": "husky install",
+    "husky:commit-msg": "commitlint -e",
+    "husky:pre-commit": "yarn lint"
```

Comment:
Just an idea, isn't pre-push a better place, in case you want to quickly submit some dirty code to switch branches? After all, we want to avoid the central build from failing and that will only happen after a push.

Reply:
pre-push is better